components/Common/CrossLink/index.module.scss

```diff
@@ -0,0 +1,25 @@
+.crossLink {
+  @apply flex flex-col items-start p-3 gap-2 bg-white dark:bg-black
```

Comment:
```suggestion
  @apply flex flex-col items-start p-3 gap-2 bg-white dark:bg-neutral-950
```

Reply:
Figma uses `neutral-950` color. I know that they are the same hex now, but in the future, one of the two may change and we will have an inconsistency.

Reply:
Good point but, there's no `neutral-950` in the tailwind [config](https://github.com/nodejs/nodejs.org/blob/main/tailwind.config.ts#L23-L33).

Reply:
it makes sense to me

Reply:
In my point of view we should keep `black`. `neutral-950` is a Sigma style, `black` is the tailwind style form "color palet"

Reply:
perhaps we ping @haydenbleasel here

Reply:
I'd rather call it neutral-950 and follow the figmas; Not sure if pinging Haydeen here is necessary as this is an implementation detail and not a design one.

Reply:
I think I'll add `neutral-950` in this PR (since it's very small changes) to make it consistent with Figma.